app/src/main/java/com/zulip/android/AsyncGetEvents.java

```diff
@@ -21,6 +21,8 @@
 import com.j256.ormlite.misc.TransactionManager;
 
 public class AsyncGetEvents extends Thread {
+    public static final String ASYNC_GET_EVENTS = "asyncGetEvents";
+    public static final String POINTER = "pointer";
```

Comment:
If these are only used internally to this class, it's a good practice to make their visibility `private`.